apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/Numeric.cs

```diff
@@ -24,21 +24,16 @@ namespace Google.Cloud.BigQuery.V2
 {
     // Note: These types are internal and used for implementing BigQueryNumeric and
     // BigQueryBigNumeric only, which are thoroughly tested.
-    // If we ever need to make these types public, then we need to test them on its own.
+    // If we ever need to make these types public, then we need to test them on their own.
     internal sealed class NumericType
     {
         private sealed class Parameters
         {
-            internal Parameters(BigInteger scaledMinValue, BigInteger scaledMaxValue, int scale)
-            {
-                GaxPreconditions.CheckArgumentRange(scale, nameof(scale), 0, PowersOf10.Length - 1);
-                GaxPreconditions.CheckState(scaledMinValue <= scaledMaxValue,
-                    $"{nameof(scaledMinValue)}={scaledMinValue} is greater than {nameof(scaledMaxValue)}={scaledMaxValue}.");
-
-                Scale = scale;
-                ScaledMinValue = scaledMinValue;
-                ScaledMaxValue = scaledMaxValue;
-            }
+            internal Parameters(BigInteger scaledMinValue, BigInteger scaledMaxValue, int scale) =>
+                // Note that we don't validate here.
+                // Instead we validate when we dedine a numeric type for the first time before adding
```

Comment:
Nit: define instead of dedine, but only bother if you're making other changes :)